openapi/jira.servicemanagement/client.bal

```diff
@@ -0,0 +1,827 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the year to 2022. Check all possible places

Reply:
Updated. Thank you